Chat_Log/Events/ManagerSave.cs

```diff
@@ -0,0 +1,14 @@
+using BrokeProtocol.API;
+using BrokeProtocol.Managers;
+
+namespace Chat_Log.Events
+{
+    public class ManagerSave : IScript
+    {
+        [Target(GameSourceEvent.ManagerSave, ExecutionMode.Event)]
+        public void OnManagerSave(SvManager manager)
+        {
+            Core.Instance.SaveSettings();
+        }
+    }
+}
```

Comment:
This will save the settings file when the server saves in case you want to edit from the game it while it's running.